core/modules/SConscript

```diff
@@ -194,8 +194,12 @@ shlibs = {}
 shlibs["qserv_common"] = dict(mods="""global memman proto mysql sql util""",
                               libs="""log protobuf mysqlclient_r boost_thread crypto""")
 
+# library implementing xrootd logging intercept (worker side)
+shlibs["xrdlog"] = dict(mods="""xrdlog""",
+                        libs="""log XrdSsiLib""")
+
 # library implementing xrootd services (worker side)
-shlibs["xrdsvc"] = dict(mods="""wbase wcontrol wconfig wdb wlog wpublish wsched xrdsvc""",
+shlibs["xrdsvc"] = dict(mods="""wbase wcontrol wconfig wdb wpublish wsched xrdsvc""",
```

Comment:
You know, I was thinking about that. Maybe we really want to keep it in some form just in case we scuttle the log4cxx back end and use the xrootd logging system that already include file rotation and the like. In that case, it would be prudent to keep this.